frontend/device/remarkable/device.lua

```diff
@@ -149,7 +149,8 @@ local f = io.open("/sys/devices/soc0/machine")
 if not f then error("missing sysfs entry for a remarkable") end
 
 local deviceType = f:read("*all") 
```

Comment:
You can probably use `*line` instead of  `*all`, that'll strip the LF for you ;).

(There might even be a `:line` method shortcut for that).